chart/kubeapps/templates/ingress.yaml

```diff
@@ -32,7 +28,7 @@ spec:
         paths:
         - path: {{ default "/" .path }}
           backend:
-            serviceName: {{ template "kubeapps.fullname" $ }}
+            serviceName: {{ template "common.names.fullname" $ }}
             servicePort: http
```

Comment:
Not a chart expert at all. However, I think there are more changes here if we use `networking.k8s.io/v1` instead of `extensions/v1beta1`: https://github.com/kubeapps/kubeapps/issues/2169#issuecomment-733556537

Reply:
Yes, let me the rest of refactorization for ingress.